scripts/release.js

```diff
@@ -15,10 +12,7 @@ yargs
     })
 
     try {
-      const version = (await fse.readJSON(path.join('package.json'))).version
-      const newVersion = semver.inc(version, argv.releaseType)
-
-      const bumpCommand = `version --new-version ${newVersion} --no-git-tag-version`
+      const bumpCommand = `version ${argv.releaseType}`
       await execute(`yarn ${bumpCommand}`, undefined, { silent: true })
```

Comment:
Here I'm using the yarn `version` plugin to bump the packages versions (See: https://yarnpkg.com/cli/version)